src/tests/models/database/sequelize/sequelizeConnector.js

```diff
@@ -178,12 +194,34 @@ function sanitizeDslResult(data) {
     return result;
 }
 
+async function saveFile(id, file) {
+    const fileClient = client.model('file');
+    let params = {
+        id: id,
+        file: file
+    };
+
+    const result = fileClient.create(params);
+    return result;
+}
+
+async function getFile(id) {
+    const fileClient = client.model('file');
+    const options = {
+        attributes: { exclude: ['updated_at', 'created_at'] }
+    };
+    options.where = { id: id };
+    const dbResult = await fileClient.find(options);
```

Comment:
.find is deprecated in sequelize, use .findOne
http://docs.sequelizejs.com/manual/models-usage.html#-code-find--code----search-for-one-specific-element-in-the-database